docs/start/getting-started/fragments/android/add-api.md

```diff
@@ -14,12 +14,22 @@ Now that your have DataStore persisting data locally, in the next step you'll co
 
 1. Run the task. You can do this by pressing the **play button** or pressing **Control-R**.
 
-1. Modify your initialization code to initialize API in order to connect to the backend. Open `MainActivity` and remove all of the previous code you entered that saved and queried for Todo items. Now, add the following code to the bottom of the `onCreate()` method:
+1. Modify your initialization code to initialize API in order to connect to the backend. Open `MainActivity`, and remove all of the code you added to `onCreate`. Replace it with the following:
```

Comment:
```suggestion
1. Modify your initialization code so that the DataStore can sync with the backend through an **API**. Open `MainActivity`, and remove all of the code you added to `onCreate`. Replace it with the following:
```